src/components/BaseCommentForm.js

```diff
@@ -109,6 +151,36 @@ class BaseCommentForm extends React.Component {
             value={this.state.commentText}
             onChange={this.handleTextChange.bind(this)}
           />
+          <FormGroup className="comment-form__file">
+            <ControlLabel><FormattedMessage id="add_images"/></ControlLabel>
+            <div className="comment-form__select-button">
+              <label className="btn btn-default btn-sm" htmlFor="fileInput">
+                <FormattedMessage id="choose_images"/>
+              </label>
+              <input
+                 type="file"
+                 ref="images"
+                 id="fileInput"
+                 multiple
+                 style={{display: 'none', visibility: 'hidden'}}
+                 onChange={(e) => this.imageTooBig(e.target.files)}
+              />
+              <div className="comment-form__selected-images">
+                {this.state.imageTooBig
+                  ? <div className="comment-form__image-too-big">
+                    <FormattedMessage id="image_too_big"/>
+                  </div>
+                  : null}
+                {this.refs.images
+                  ? this.getSelectedImagesAsArray(this.refs.images.files).map(
+                      (image) =>
+                        <p style={image.size > 1000000 ? {color: 'red'} : null}>
+                          {image.name} {round((image.size / 1000000), 3) + 'MB'}
```

Comment:
Wonder if we should actually display the image thumbnails instead of the image names here? So the user will see their comment the same way as after it is posted? Also, I would render the thumbnails next to the button (not below), so the form does not get any taller than necessary.